pkg/sriov/network.go

```diff
@@ -418,3 +418,31 @@ func (builder *NetworkBuilder) validate() (bool, error) {
 
 	return true, nil
 }
+
+// CleanAllNetworksByTargetNamespace deletes all networks matched by their NetworkNamespace spec.
+func CleanAllNetworksByTargetNamespace(
+	apiClient *clients.Settings,
+	operatornsname string,
+	targetnsname string,
```

Comment:
Please check that parameter is not empty

Reply:
Added check.